src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -881,7 +881,22 @@ public IEnumerable<UnspentOutputReference> GetSpendableTransactionsInWallet(stri
 
         public virtual IEnumerable<UnspentOutputReference> GetSpendableTransactionsInWalletForStaking(string walletName, int confirmations = 0)
         {
-            return this.GetSpendableTransactionsInWallet(walletName, confirmations);
+            return this.GetUnspentTransactionsInWallet(walletName, confirmations, Wallet.NormalAccounts);
```

Comment:
Isn't this a change in the logic of this method?
Perhaps its ok if its only used by the staking feature